src/main/kotlin/apply/ui/admin/selections/GradeEvaluationForm.kt

```diff
@@ -0,0 +1,62 @@
+package apply.ui.admin.selections
+
+import apply.application.GradeEvaluationData
+import apply.application.GradeEvaluationItemData
+import apply.domain.evaluationtarget.EvaluationStatus
+import com.vaadin.flow.component.select.Select
+import com.vaadin.flow.component.textfield.IntegerField
+import com.vaadin.flow.component.textfield.TextArea
+import support.views.BindingFormLayout
+
+class GradeEvaluationForm :
+    BindingFormLayout<GradeEvaluationData>(GradeEvaluationData::class) {
+
+    private val gradeEvaluationItems: MutableList<GradeEvaluationItemForm> = mutableListOf()
```

Comment:
공백 라인을 의미 있게 사용하는 것이 좋아 보이네요.

```suggestion
class GradeEvaluationForm : BindingFormLayout<GradeEvaluationData>(GradeEvaluationData::class) {
    private val gradeEvaluationItems: MutableList<GradeEvaluationItemForm> = mutableListOf()
```